pandas/util/_decorators.py

```diff
@@ -65,8 +65,9 @@ def deprecate_kwarg(old_arg_name, new_arg_name, mapping=None, stacklevel=2):
     ----------
     old_arg_name : str
         Name of argument in function to deprecate
-    new_arg_name : str
-        Name of preferred argument in function
+    new_arg_name : str | None
```

Comment:
Usually this is written as `str or None`.

Reply:
Thanks for catching this -- didn't realize this is the case.  Please refer to 4c83659350328784ecdac963f7bb8b6a5acbf577